controller/controller.ts

```diff
@@ -0,0 +1,45 @@
+import User from "../models/model";
+
+export const getUsers = async (): Promise<string> => {
+  try {
+    return await User.getAll();
+  } catch (err) {
+    throw err;
+  }
+};
+
+export const postUser = async (
+  id: number,
+  name: string,
+  age: string
+): Promise<string> => {
+  const user = new User(id, name, age);
+  try {
+    return await user.addUser();
+  } catch (err) {
+    throw err;
+  }
```

Comment:
Look at project structure https://github.com/hagopj13/node-express-boilerplate.
The controller should be responsible for returning the http response to the user.

Reply:
Changed